src/main/java/de/retest/web/WebElementPeer.java

```diff
@@ -86,17 +86,22 @@ public OutlineAttribute retrieveOutline() {
 			return null;
 		}
 		try {
-			final int x = Integer.parseInt( webData.get( AttributesConfig.X ) );
-			final int y = Integer.parseInt( webData.get( AttributesConfig.Y ) );
-			final int width = Integer.parseInt( webData.get( AttributesConfig.WIDTH ) );
-			final int height = Integer.parseInt( webData.get( AttributesConfig.HEIGHT ) );
+			final int x = toInt( webData.get( AttributesConfig.X ) );
+			final int y = toInt( webData.get( AttributesConfig.Y ) );
+			final int width = toInt( webData.get( AttributesConfig.WIDTH ) );
+			final int height = toInt( webData.get( AttributesConfig.HEIGHT ) );
 			return new OutlineAttribute( new Rectangle( x, y, width, height ) );
 		} catch ( final Exception e ) {
 			logger.error( "Exception retrieving outline: ", e );
 		}
 		return null;
 	}
 
+	private static int toInt( final String input ) {
+		// TODO java.lang.NumberFormatException: For input string: "auto"
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=40&issues=AWT1saZP81b9c4_ZdBaY&open=AWT1saZP81b9c4_ZdBaY)

[ISSUE_KEY]: AWT1saZP81b9c4_ZdBaY

Reply:
You could use `NumberUtils.toInt`